src/App.js

```diff
@@ -22,7 +22,7 @@ function App(props) {
   // ^Needed to stop the screen from scrolling when the modal is visible!
 
   return (
-    <div className="App" style={{height: "100%", "overflow-y": isDisplaying ? "hidden" : "visible" }}>
+    <div className="App" style={{height: "100%", "overflowY": isDisplaying ? "hidden" : "visible" }}>
```

Comment:
is this overflowY a prop?

Reply:
was one of the errors I was getting. doesn't even make sense that the key has double quotes